src/components/shoppingListGrid.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Card,
+  CardContent,
+  CardActions,
+  CardActionArea,
+  Button,
+  Typography,
+  Grid,
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: 350,
+    marginBottom: 35,
+  },
+  cardActionArea: {
+    height: 75,
+  },
+});
+
+const ShoppingListGrid: React.FC = () => {
+  const classes = useStyles();
+  const shoppingCards = [
+    { shoppingListName: 'Shopping List 1' },
```

Comment:
Add a `// TODO: This is just mock data with a name, add real data later` comment here